ui/src/Logout.tsx

```diff
@@ -25,7 +25,7 @@ export class Logout extends PureComponent<Props> {
 
   private handleSignOut = async () => {
     if (CLOUD) {
-      window.location.pathname = CLOUD_SIGNOUT_PATHNAME
+      window.location.href = CLOUD_SIGNOUT_URL
```

Comment:
Should we also update the following: https://github.com/influxdata/influxdb/blob/916f74569a479fe59a930b031b290411df12844c/ui/src/Signin.tsx#L81